metrics-core/src/main/java/com/yammer/metrics/core/MetricsRegistry.java

```diff
@@ -25,11 +25,8 @@ public class MetricsRegistry {
 
     private static final Function<Boolean, Sample> DEFAULT_HISTOGRAM_SAMPLE_FUNCTION =
       (
-        biased -> {
-            Sample baseSample = biased
-              ? SampleType.BIASED.newSample()
-              : SampleType.UNIFORM.newSample();
-            return new SummaryAndBucketSample(baseSample, new VictoriaBucketSample());
```

Comment:
Safe to delete `SummaryAndBucketSample` entirely?

Reply:
eventually yes, I am going to do that in sep PR once I confirm this does everything I want.